packages/core/helper-plugin/lib/src/utils/getAPIInnerErrors/index.js

```diff
@@ -1,16 +1,28 @@
+import { normalizeAPIError } from '../../hooks/useAPIErrorHandler';
+
+/**
+ *
+ * Returns a normalized error message
+ *
+ * @deprecated
+ * @export
+ * @param {error} error - API error response object
+ * @param {object<{ getTrad }>} - Error message prefix callback
+ * @return {object}
+ */
```

Comment:
Before you say anything Josh: these are temporarily and if you agree with the overall idea I'll beautify them :p

Reply:
hahaha I'd rather for the return you used a `Record<string, string>` 👀 and the `getTrad` param i think you can omit `object` but keep `{ getTrad }`